src/classifications/scheduler.ts

```diff
@@ -95,23 +95,27 @@ export default class ClassificationsScheduler extends CommandBase {
 
   pollForCompletion = (id: any): Promise<Classification> => {
```

Comment:
Why does classifcation have its own timeout check?

Reply:
This has been in the client for a long time. This is the implementation of an option which tells the client to block until classification is complete.